battleship-server/src/main/java/com.epam.solid.nie/server/CommunicationProtocol.java

```diff
@@ -2,7 +2,7 @@
 
 public class CommunicationProtocol implements Protocol {
     @Override
-    public String processInput(String inputString) {
+    public String processInput(String inputString, int id) {
```

Comment:
what id stands for? change name to something more meaningful / or write java doc for this method

Reply:
removed in last commit